---
simple_prediction_node:
  ros__parameters:
    prediction_horizon: 3.0    # seconds into the future
    prediction_time_step: 0.2  # seconds between predicted poses
